applications/TrilinosApplication/trilinos_space.h

```diff
@@ -223,6 +224,12 @@ class TrilinosSpace
         rA.Multiply(transpose_flag, rX, rY);
     }
 
+    static void Mult(const MatrixType& rA, const MatrixType& rB, MatrixType& rC, const bool TransposeA=false, const bool TransposeB=false)
+    {
+        // rC = rA*rB
+        EpetraExt::MatrixMatrix::Multiply(rA, TransposeA, rB, TransposeB, rC);
```

Comment:
I am afraid this is not that simple. This multiply operation is complicated in Trilinos/Epetra. I think we have to do some checks before we pass the matrices to Epetra/Trilinos. Otherwise, this will result in a weird errors which can be quite hard to track. 

Secondly, I think we should check the error which is returned by this Multiply function of Epetra and throw some meaning full errors from Kratos.